### PR TITLE
add-job-logger-for-signals

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

**Comment by fviale** [comment]
Retest this please

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java [hunk 1]
```diff
@@ -1876,10 +1876,12 @@ private JobInfo insertJobSignals(JobInfo jobInfo) {
         if (checkJobPermissionMethod(jobid, "addJobSignal")) {
 
             try {
-                if (publicStore.channelExists(SIGNAL_ORIGINATOR, SIGNAL_TASK_ID, signalsChannel + jobid)) {
+                JobId jobIdObj = JobIdImpl.makeJobId(jobInfo.getJobId().value());
+                TaskId taskId = TaskIdImpl.createTaskId(jobIdObj, "notask", -1);
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java:1880]: `"notask"` should be extracted as a constant

Comment (fviale) [reply scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java:1880]: the two lines could be extracted as a private method

**Review by fviale — APPROVED** [review]
minor comments, otherwise LGTM

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java [hunk 2]
```diff
@@ -1668,15 +1670,21 @@ private void insertVisualization(List<JobInfo> jobsInfo) {
...
-                                          SIGNAL_TASK_ID,
+                                          taskId,
                                           signalsChannel + jobInfo.getJobId().value()) != null;
             } catch (InvalidChannelException e) {
             }
             return false;
         }).forEach(jobInfo -> insertJobSignals(jobInfo));
     }
 
+    private TaskId getNoTaskId(String value) {
```
Comment (fviale) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java:1683]: the parameter should be named jobId

the method name should be createFakeTaskId

**Comment by fviale** [comment]
Retest this please

**Review by marounkoussaifi — APPROVED** [review]

**Pull request merged**